website/src/views/conceptMap.js

```diff
@@ -11,7 +11,17 @@ export default function ConceptMap() {
     const [loading, setLoading] = useState(false);
     const [studentMastery, setStudentMastery] = useState('000000');
 
-    const { selectedStudent, setSelectedStudent } = useContext(StudentSelectionContext);
+    const iframeRef = useRef(null);
```

Comment:
what does setting to null do - just curious

Reply:
That just initializes the reference to null.  Note that the hooks necessarily load before the html/css meaning when we initialize the ref, we don't yet know what it is referring to.  Once we load the component we are rendering (this is also called mounting the component), we now have a value to assign to the ref.  We do this by assigning the special data tag "ref" on the element and assigning it to the value of the ref we want to assign it to.